unit-tests/py/rspy/devices.py

```diff
@@ -186,26 +186,39 @@ def by_name( name ):
     return { device.serial_number for device in _device_by_sn.values() if device.name  and  device.name.find( name ) >= 0 }
 
 
+def _get_sns_from_spec( spec ):
+    """
+    helper function for by_configuration. Yields all devices matching the given spec
+    """
+    if spec.endswith( '*' ):
+        # By product line
+        for sn in by_product_line( spec[:-1] ):
+            yield sn
+    else:
+        # By name
+        for sn in by_name( spec ):
+            yield sn
+
+
 def by_configuration( config ):
     """
     :param config: A test:device line collection of arguments (e.g., [L515 D400*])
-    :return: A set of device serial-numbers matching
+    :return: A set of device serial-numbers matching. One matching serial number for every device in config.
```

Comment:
It doesn't return anything any more

Reply:
Do you want me to change the `:return:` to `:yield:` or just remove it?